### DISCUSSION TITLE
Support for edgesIgnoringSafeArea in Android

### DISCUSSION THREAD

**Discussion by Saksham-245** [skiptools/skip#166] [General]
Hi All,

Currently I am trying out skip by building an to do app and in that I am to apply colour to whole background using ZStack and in that applying `Color(red: 248, green: 238, blue: 226).edgesIgnoringSafeArea(Edge.Set.all)` but on Android it is making the app crashed.

**Comment by aabewhite** [comment]
Thanks for the report! .edgesIgnoringSafeArea has been deprecated in SwiftUI, so Skip does not support it. Please try with .ignoresSafeArea:

https://developer.apple.com/documentation/swiftui/view/ignoressafearea(_:edges:)

Let us know if you have any trouble with this API.

**Reply by Saksham-245** [reply]
Ok Thanks I will try this!

**Comment by Saksham-245** [comment]
Android only crashing if applying on this rgb colour values like this Color(red: 248, green: 238, blue: 226) but not on like Color.red or Color(hue: 0.9, saturation: 0.8, brightness: 0.9, opacity: 1)

**Reply by aabewhite** [reply]
The SwiftUI Color(red:green:blue:opacity:) constructor is supposed to normally take values between 0 and 1, not values between 0 and 255.

It shouldn't cause a crash, though, so I'll make sure to update our internal code to clamp the given values in an upcoming release.

**Reply by Saksham-245** [reply]
Ok Sure!

**Reply by Saksham-245** [reply]
Finally got it working with the color I want by using this Color(red: 0.9725, green: 0.9333, blue: 0.8863).ignoresSafeArea()
